lib/zustr2stp.h

```diff
@@ -14,6 +14,10 @@
 #include <string.h>
 
 #include "mempcpy.h"
+#include "sizeof.h"
+
+
+#define ZUSTR2STP(dst, src)  zustr2stp(dst, src, SIZEOF_ARRAY(src))
```

Comment:
No.  This macro receives the size of the **source** buffer.  It assumes the destination buffer will be large enough, which means at least sz+1.

I designed zustr2stp() that way, because since it's the source that is usually an array, you can call it using SIZEOF_ARRAY(), and forget about doing +1, which is always error-prone.